Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppDeploy/MsDeployOnTargetMachines.ps1

```diff
@@ -77,7 +81,25 @@ function Get-MsDeployCmdArgs
         throw "Package does not exist : `"$webDeployPackage`""
     }
 
-    $msDeployCmdArgs = [string]::Empty
+    $msDeployCmdArgs = [String]::Format(" -verb:sync")
+
+    if($isFolderBasedDeployment)
+    {
+        $msDeployCmdArgs += [String]::Format(' -source:iisApp="{0}"', $webDeployPackage);
+        $msDeployCmdArgs += [String]::Format(' -dest:iisApp="{0}"', $websiteName);
+    }
+    else
+    {
+        $msDeployCmdArgs += [String]::Format(' -source:package="{0}"', $webDeployPackage);
+        if($isParamFilePresentInPacakge)
+        {
+            $msDeployCmdArgs += [String]::Format(' -dest:auto');
+        }
+        else {
+            $msDeployCmdArgs += [String]::Format(' -dest:contentPath="{0}",', $websiteName);
```

Comment:
Verify zip and package created from MsDeploy with source:iisApp and dest:iisApp is working if we change the dest:IisApp in 99

https://github.com/Microsoft/vsts-rm-extensions/issues/120